### PR TITLE
Add small Whimsylib changes from implementing Office floor

### PR DESCRIPTION
**Pull request by abjose**

### CODE REVIEW

**Comment by abjose** [comment]
@xmunoz @flosincapite

**Comment by xmunoz** [comment]
Can you add us as reviewers @abjose ?

**Comment by abjose** [comment]
I can't for some reason :( at least, I don't see the little gear widget next to "Reviewers"

**Comment by flosincapite** [comment]
Looks like it worked somehow?

**Comment by flosincapite** [comment]
Oh, whoops, wrong PR. Okay, I gave you maintainer permissions on whimsylib,
so that should fix it.

File: whimsylib/actor.py [hunk 1]
```diff
@@ -170,6 +170,12 @@ def __init__(self, actor_item, *statistics, **kwargs):
     def ai(self):
         return self._ai
 
+    @ai.setter
+    def ai(self, new_ai):
```
Comment (xmunoz) [inline whimsylib/actor.py:172]: Should you also call this above on line [155](https://github.com/Cronenbrogues/whimsylib/pull/11/files#diff-412ce60c55f26cf5fc67b150ecf98ba84b6d4c2dc23398425fada9d5badb7a54R155)?

File: whimsylib/actor.py [hunk 2]
```diff
@@ -184,6 +190,10 @@ def abilities(self):
     def idle_text(self):
         return self._idle_text
 
+    @idle_text.setter
```
Comment (xmunoz) [inline whimsylib/actor.py:191]: Same for line 158.

**Comment by xmunoz** [comment]
LGTM, just a question about calling these setter methods in the Actor `__init__` method.

**Review by xmunoz — APPROVED** [review]

**Comment by abjose** [comment]
Thanks for the review!

**Pull request merged**